.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Go
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test-build:
+    name: Test build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
```

Comment:
Is it possible to do a matrix of go versions with github actions?

Reply:
Yes, it's possible.